snakegame.py

```diff
@@ -94,23 +98,43 @@ def rungame(self, init_length, init_speed, fruits_no):
 		print("Runing new game.")
 
 		# Game init
-		snake = Snake(3, 500, 500)
+		snake = Snake()
 		snake.joints = [Joint(5,8), Joint(4,8), Joint(3,8)]
 		fruits = []
 		for _ in range(fruits_no):
-			fruits.append(Joint(random.randrange(16), random.randrange(16)))
+			fruits.append(Joint(random.randrange(self.fieldSizeX), random.randrange(self.fieldSizeY)))
+		walls = []
```

Comment:
I am wondering whether this `walls` and `snake` variables should be method variables or class variables with `self.`

Only benefit of keeping it as a method variable would be that we could call this method `rungame()` multiple times and the environment would be isolated. But - we can not do this right now, because `rungame()` is dependent on `self.running`.

Thus I would suggest to use `self.walls` instead of `walls`. What do you think?

Reply:
It's a good notice. I think encapsulation within `rungame` method is more usable for us right now, because there is no need to access these variables from other parts of code. It leads me to think opposite way about `self.running` now. I would say, we`d better encapsulate this variable within the method as well. Because we do not touch it from nowhere else of the code.
However! If we bring new design at run game level, there might be need of writing as you suggest. There is no menu implemented yet.

Reply:
Ok this sounds good. Go on with encapsulation of `running`